### ISSUE TITLE
Error couldn't connect SSH: ssh: handshake failed: ssh: unable to authenticate

### ISSUE THREAD

**Issue by Kipjr** [rclone/rclone#5369]
#### The associated forum post URL from `https://forum.rclone.org`

n/a

#### What is the problem you are having with rclone?

viewing in GUI the contents of remote or viewing it using cli

rclone    | 2021/05/30 16:16:39 ERROR : rc: "operations/list": error: NewFs: couldn't connect SSH: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain

using SSH on the docker host or using Filezilla does work. 

#### What is your rclone version (output from `rclone version`)

rclone v1.56.0-beta.5530.df60e6323
- os/version: alpine 3.13.5 (64 bit)
- os/kernel: 5.4.51-v8+ (aarch64)
- os/type: linux
- os/arch: arm64
- go/version: go1.16.4
- go/linking: static
- go/tags: none

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

_Raspbian 64 bit with Docker and image: rclone/rclone:beta_

#### Which cloud storage system are you using? (e.g. Google Drive)

sftp: https://www.transip.nl/stack/ 
https://www.transip.nl/knowledgebase/artikel/2083-sftp-voor-stack/

[stack]
type = sftp
host = ACCOUNT.stackstorage.com
pass = thisisnotmypassword
user = USERNAME@ACCOUNT
known_hosts_file = /config/rclone/ssh_known_hosts

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

rclone tree stack:/ -vv

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

2021/05/30 16:34:15 DEBUG : rclone: Version "v1.56.0-beta.5530.df60e6323" starting with parameters ["rclone" "tree" "stack:/" "-vv"]
2021/05/30 16:34:15 DEBUG : Creating backend with remote "stack:/"
2021/05/30 16:34:15 DEBUG : Using config file from "/config/rclone/rclone.conf"
2021/05/30 16:34:16 DEBUG : pacer: low level retry 1/10 (error couldn't connect SSH: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain)
2021/05/30 16:34:16 DEBUG : pacer: Rate limited, increasing sleep to 200ms

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
> pass = thisisnotmypassword

Is the password in your config file obscured? That needs to be the output of

```
$ rclone obscure thisisnotmypassword
LNTT4Jc_tTTSn7so_trtmX1KDLSq528lowl0PS7ysXRGkNA
```

so

> pass = LNTT4Jc_tTTSn7so_trtmX1KDLSq528lowl0PS7ysXRGkNA

**Comment by Kipjr** [comment]
It works, thanks! When I added the credentials in the GUI they are not hashed/encrypted. Shouldn't that be a bug?

**Comment by ncw** [comment]
> It works, thanks! When I added the credentials in the GUI they are not hashed/encrypted. Shouldn't that be a bug?

Yes that sounds like a bug! Can you report it in the GUI issues [new issue](https://github.com/rclone/rclone-webui-react/issues/new/choose) on Github.

**Comment by ncw** [comment]
Actually, I'll move this issue to the GUI project - don't make a new one!
